### DISCUSSION TITLE
Marketplace sort on most installs/stars has no effect

### DISCUSSION THREAD

**Discussion by wkillerud** [community/community#126395] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The result seems to be identical to Best match. The sorting is not based on number of stars/installs.

https://github.com/community/community/assets/1223410/7849f6ee-b0ca-44a2-a2a9-8d25b868dd4a

**Comment by gil-son** [comment]
Hello, how are you?


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!
